NFIQ2/NFIQ2Algorithm/src/NFIQ2Algorithm.cpp

```diff
@@ -35,6 +35,18 @@ NFIQ::NFIQ2Algorithm::computeQualityScore(
              qualityFeatureSpeed ) );
 }
 
+std::list<NFIQ::QualityFeatureData> NFIQ::NFIQ2Algorithm::computeQualityFeatures(
+  const NFIQ::FingerprintImageData& rawImage )
+{
+	std::list<NFIQ::ActionableQualityFeedback> actionableQuality;
+	std::list<NFIQ::QualityFeatureSpeed> qualityFeatureSpeed;
+  
+	return ( this->pimpl->computeQualityFeatures(
+			rawImage,
+			false, actionableQuality,
```

Comment:
Why should these `bool`s always be `false`? Why not keep exposing the parameters as it is in the implementation? I get that you don't need them for what you want to do, but someone else might. They should be kept.

Reply:
for the training we don't need the additional information, so I omitted them in the top level implementation, but it will be ok for me if we expose these values also.